pyvrp/stop/tests/test_MultipleCriteria.py

```diff
@@ -0,0 +1,49 @@
+from numpy.testing import assert_, assert_raises
+from pytest import mark
+
+from pyvrp.stop import MaxRuntime, MultipleCriteria, NoImprovement
+from pyvrp.tests.helpers import sleep
+
+
+def test_raises_if_empty():
+    """
+    Tests that passing no criteria results in an error.
+    """
+    with assert_raises(ValueError):
+        MultipleCriteria([])
+
+
+def test_stops_if_zero_max_iterations():
+    """
+    Tests that the stopping criterion stops immediately if zero maximum
+    iterations are allowed.
+    """
+    stop = MultipleCriteria([NoImprovement(0), MaxRuntime(0.100)])
```

Comment:
```suggestion
    stop = MultipleCriteria([MaxIterations(0), MaxRuntime(0.100)])
```

Suggestion: It's a bit confusing to write max iterations but then to use the no improvement criterion. I cannot see the direct benefit of using no improvement over max iterations so I suggest replacing it (here and in the other tests).

Reply:
Fair. I copied these tests from the old `TimedNoImprovement`. I'll update this now.